src/matrix/json.rs

```diff
@@ -17,6 +17,7 @@
 use rustc_serialize::json::{Json,Array};
 
 pub fn path<'a>(json: &'a Json, path: &str) -> &'a Json {
+    // why not just use json.find_path ooi? -matthew
```

Comment:
wanted to quickly prototype json parsing without a billion .unwrap().unwrap().unwrap() calls. no reason to not use it at this point.